### PR TITLE
Installer Fix

### PR DESCRIPTION
**Pull request by aucsta**
This fixes server error when trying to load installer page which doesn't appear when using a virtual server such as WAMP or MAMP it only happens when using a proper server

### CODE REVIEW

File: includes/database/DatabasePDO.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ class DatabasePDO extends Database
         'FETCH_BOTH' => PDO::FETCH_BOTH,
         'FETCH_NUM' => PDO::FETCH_NUM,
     ];
-    protected $error_supress = !WeBidDebug;
+    protected $error_supress = WeBidDebug;
```
Comment (renlok) [inline includes/database/DatabasePDO.php]: The problem with this is it now changes the definitions of WeBidDebug so they no long make sense
WeBidDebug = true will disable debug mode

Think the best solution is to move the definition into the class creator

Comment (aucsta) [inline includes/database/DatabasePDO.php]: Do you know how to do this because I have a clue

Comment (aucsta) [inline includes/database/DatabasePDO.php]: This is the error I get when trying to install. Parse error: syntax error, unexpected '!' in /hermes/bosnaweb07a/b1189/ipg.aucstacom/includes/database/DatabasePDO.php on line 26

**Comment by aucsta** [comment]
would be helpful if someone could look into this

**Comment by renlok** [comment]
My recent commit should hopefully of fixed this, can you let me know

**Comment by aucsta** [comment]
yes so far so good brilliant work. are you going to delete the top 2 issues on the issues section as they are now fixed

**Pull request closed without merging**